src/app/meeting/[id]/page.tsx

```
@@ -0,0 +1,271 @@
+  waiting: [
+    { gen: "14기", name: "장잇프" }, { gen: "14기", name: "명자동구" },
+    { gen: "15기", name: "고은성" },
+  ] as Member[],
+  noResponseCount: 6,
+};
+
+function MemberList({ members }: { members: Member[] }) {
+  return (
+    <div className="mt-4 grid grid-cols-2 gap-x-4 gap-y-3">
+      {members.map((m, i) => (
+        <div key={i} className="flex items-center gap-2">
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)